xml/System/String.xml

```diff
@@ -8217,7 +8225,9 @@ A string is empty if it  is explicitly assigned an empty string ("") or <xref:Sy
  If `separator` is `null`, an empty string (<xref:System.String.Empty?displayProperty=nameWithType>) is used instead. If any member of `values` is `null`, an empty string is used instead.  
   
  <xref:System.String.Join%28System.String%2CSystem.Collections.Generic.IEnumerable%7BSystem.String%7D%29> is a convenience method that lets you concatenate each element in an `IEnumerable(Of String)` collection without first converting the elements to a string array. It is particularly useful with Language-Integrated Query (LINQ) query expressions. The following example passes a `List(Of String)` object that contains either the uppercase or lowercase letters of the alphabet to a lambda expression that selects letters that are equal to or greater than a particular letter (which, in the example, is "M"). The `IEnumerable(Of String)` collection returned by the <xref:System.Linq.Enumerable.Where%2A?displayProperty=nameWithType> method is passed to the <xref:System.String.Join%28System.String%2CSystem.Collections.Generic.IEnumerable%7BSystem.String%7D%29> method to display the result as a single string.  
-  
+
+[!INCLUDE[interactive-note](~/includes/csharp-interactive-note-some.md)]
```

Comment:
move the note to the membergroup remarks and the example for Join(String, String[], Int32, Int32) should also be made interactive

Reply:
Done in https://github.com/dotnet/samples/pull/1171